CONTRIBUTING.md

```diff
@@ -196,6 +196,9 @@ and set up a new project.
 You can use the [metals][] Scala language server together with [lsp-mode][] for
 a nice IDE experience in the world's best lisp driven operating system.
 
+If you are developing in Emacs, you probably also want to get familiar with the
+incremental build server [bloop](./docs/internal/using-bloop.md).
```

Comment:
You should ask @konnov for a review of this, I don't use Emacs.

Reply:
It's not dependent on anyone using emacs. It's just some dev documentation needing a rubber stamp :)

Reply:
I am not actively using emacs, though I know how to quit it. It definitely helps to have the notes, since Shon has went through all the setup already.